spec/support/features/webauthn_helper.rb

```diff
@@ -98,6 +98,28 @@ def attestation_object
     HEREDOC
   end
 
+  def backed_up_authenticator_data_flags
+    {
+      up: true,
+      uv: false,
+      be: true,
+      bs: true,
+      at: true,
+      ed: false,
+    }.to_s
```

Comment:
Are these meant to be JSON? Stringifying the hash won't produce JSON, but we could use `.to_json`?